haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="A time management system"
+DESCRIPTION="Eventual is a personal time management system for Haiku. \
+Intended to be a full-featured organizer. "
+HOMEPAGE="https://sourceforge.net/projects/eventual/"
+COPYRIGHT="2011 Alexey Burshtein \
+2001, 2002 Kevin H. Patterson"
+LICENSE="MIT BSD"
```

Comment:
```diff
 COPYRIGHT="2011 Alexey Burshtein \
-2001, 2002 Kevin H. Patterson"
+	2001, 2002 Kevin H. Patterson"
-LICENSE="MIT BSD"
+LICENSE="MIT
+	BSD (3-clause)"
```
(I'm not sure if it's 2, 3, or 4-clause.)

Reply:
Whoops.
Didn't see this until now :(

Reply:
@fbrosson It seems to be 3-clause.